apps/expo/package.json

```diff
@@ -30,9 +28,12 @@
     "react-dom": "18.2.0",
     "react-native": "0.71.3",
     "react-native-safe-area-context": "4.5.0",
-    "react-native-screens": "~3.20.0"
+    "react-native-screens": "~3.20.0",
+    "superjson": "1.9.1"
   },
   "devDependencies": {
+    "@acme/api": "*",
```

Comment:
superjson can no longer be importerad from the api package as we dont share any runtime code from it